test/run-test.ps1

```diff
@@ -1,169 +1,64 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
 [cmdletbinding()]
 param(
-    [switch]$UseImageCache,
     [string]$Filter,
     [string]$Architecture
 )
 
 Set-StrictMode -Version Latest
 $ErrorActionPreference = 'Stop'
```

Comment:
Shouldn't we reset `$ErrorActionPreference` to the state before it was modified? Maybe this script is not intended to be invoked directly by a user/developer.  But `build-and-test.ps1` would be invoked directly, and that means we need to reset the preference.

Reply:
Per our conversation, ErrorActionPreference is scoped to the script in a non-ISE environment.  I don't think restoring is necessary for this usage.